relayer.go

```diff
@@ -17,14 +17,15 @@ import (
 // PROTOCOL_ID allows data identification by looking at the first few bytes
 var PROTOCOL_ID = []byte{0x72, 0x6f, 0x6c, 0x6c}
 
+const (
+	DEFAULT_SAT_AMOUNT  = 1000
+	DEFAULT_SAT_FEE     = 200
```

Comment:
[Question]
Shouldn't this be configurable? Because, if I understand right, the whole block data will be posted, not just a commitment to it. Thus, its size can change. Wouldn't it be better to have the fee specified as `vBytes` so that it changes according to the amount of data being posted?